docs/UserGuide.md

```diff
@@ -26,41 +22,53 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 i/1022` : Adds a client named `John Doe` to the ClientBook.
 
    * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
 
-   * **`clear`** : Deletes all contacts.
-
    * **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
+1. Refer to the [Features](#features) below for the full list of commands and their descriptions.
 
 --------------------------------------------------------------------------------------------------------------------
-
 ## Features
+- Viewing help : `help`
+- Adding a client: `add`
+- List all clients: `list`
+- Locating clients by name: `find`
+- Deleting a client : `delete`
+- Exiting the program : `exit`
+- Saving the data 
+
+--------------------------------------------------------------------------------------------------------------------
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Words in UPPER_CASE are the parameters to be supplied by the user.
+ e.g. in `add n/NAME`, NAME is a parameter which can be used as `add n/John Doe`.
+
+
+* Items in square brackets are optional.
+ e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Items with …​ after them can be used multiple times including zero times.
+ e.g. `[t/TAG]…​` can be used as   (i.e. 0 times), `t/basicplan`, `t/basicplan t/lifeinsurance` etc.
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+* Parameters can be in any order.
+ e.g. if the command specifies n/NAME p/PHONE_NUMBER, p/PHONE_NUMBER n/NAME is also acceptable.
```

Comment:
Would it be clearer to the user if we used markdown to format the keywords here too?